### DISCUSSION TITLE
Enabling Settings Sync deleted all local settings without merging

### DISCUSSION THREAD

**Discussion by mercExec** [microsoft/vscode-discussions#771] [Extension Development QnA] [answered]
Hello

Today i noticed i didn't have setting sync on, so i enabled it, and before it was enabled, there was a conflict with the settings file and there was a "merge" option so i clicked it, but that didn't merge anything and all my settings and extension settings were lost. I tried looking for a way to revert the changes but couldn't find anything that helped

I tried the following:
-Going to "Settings Sync: Show Synced Data > SYNC ACTIVITY (LOCAL)" but it was completely empty.
-Logging out & in of github account
-Going to the backup folder. Nothing is there but the synced data (which is the default data)

Is there another way to revert changes to the settings.json file? That's the only important one i need at the moment. Also, I'm sorry if this isn't the right place to ask this question.

**Comment by gjsjohnmurray** [comment, marked as answer]
This forum is intended for developers who are creating VS Code extensions.

Please raise your issue at https://github.com/microsoft/vscode/issues

**Reply by mercExec** [reply]
Noted. Thank you.
